tensorflow_recommenders_addons/BUILD

```diff
@@ -15,12 +15,13 @@ config_setting(
     constraint_values = ["@bazel_tools//platforms:windows"],
 )
 
+# Disable EV due to a required header file is removed from Tensorflow 2.13
 py_library(
     name = "tensorflow_recommenders_addons",
     srcs = glob(["**/*.py"]),
     deps = [
         "//tensorflow_recommenders_addons/dynamic_embedding/python/ops:dynamic_embedding",
-        "//tensorflow_recommenders_addons/embedding_variable/python:embedding_variable",
+        # "//tensorflow_recommenders_addons/embedding_variable/python:embedding_variable",
```

Comment:
It's recommended that all of the comments be changed to remove directly, we can revert is needed in the future.

Reply:
Removed